src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -217,28 +218,44 @@ export default (inflictor, value, type, ptooltip) => {
         <div className="inflictorWithValue" data-tip={tooltip && true} data-for={ttId}>
           {!type && <img src={image} alt="" />}
           {type === 'buff' &&
-          <div
-            className="buff"
-            style={{
-              backgroundImage: `url(${image})`,
-            }}
-          />
+            <div
+              className="buff"
+              style={{
+                backgroundImage: `url(${image})`,
+              }}
+            />
           }
           {!type && <div className="overlay">{value}</div>}
           {type === 'buff' &&
-          <div className="buffOverlay">
-            {value > 0 && value}
-          </div>
+            <div className="buffOverlay">
+              {value > 0 && value}
+            </div>
           }
           {tooltip &&
-          <div className="tooltip">
-            <ReactTooltip id={ttId} effect="solid" place="left">
-              {tooltip}
-            </ReactTooltip>
-          </div>}
+            <div className="tooltip">
+              <ReactTooltip id={ttId} effect="solid" place="left">
+                {tooltip}
+              </ReactTooltip>
+            </div>}
         </div>
       </StyledDiv>
     );
   }
   return null;
 };
+
+InflictorWithValueComp.propTypes = {
+  inflictor: PropTypes.string,
+  value: PropTypes.string,
+  type: PropTypes.string,
+  ptooltip: PropTypes.shape({}),
+  abilities: PropTypes.shape({}),
+};
+
+const mapStateToProps = state => ({
+  abilities: state.app.abilities.data || {},
```

Comment:
don't assign a default object here, this causes react to re-render the component all the time